pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -56,7 +56,7 @@ autoPatchelf() {
     local runtimeDependenciesArray=($runtimeDependencies)
     @pythonInterpreter@ @autoPatchelfScript@                            \
         ${norecurse:+--no-recurse}                                      \
-        ${autoPatchelfIgnoreMissingDeps:+--ignore-missing}              \
+        --ignore-missing "${autoPatchelfIgnoreMissingDeps[@]}"          \
```

Comment:
This is about the right place to also support `autoPatchelfIgnoreMissingDeps = true` format, for backward compatibility. We may have some code out there that relies on that feature. Not supporting booleans is a breaking change, and addinc some code to handle `true -> [ '*' ]` and `false -> [ ]` is a small price to pay.